### PR TITLE
Fix #180841 enterprisezine.jp

### LINKED ISSUE #180841: enterprisezine.jp
### Issue URL (Social Widget)
 [https://enterprisezine.jp/article/detail/19599](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fenterprisezine.jp%2Farticle%2Fdetail%2F19599)
Username: `@White00100`

### Screenshots
<details>
  <summary>Screenshot 1 </summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard for Android v4.5 Nightly 3
System version: | Android 14
Browser: | Chrome
License type: | paid
AdGuard mode: | VPN
HTTPS filtering: | enabled
Stealth mode options: | Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Self-destructing third-party cookies (180), <br>Self-destructing first-party cookies (4320)
DNS filtering: | server: `System`<br>
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Popups, <br/>AdGuard Widgets, <br/>AdGuard Mobile App Banners, <br/>AdGuard Cookie Notices, <br/>AdGuard Other Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese
Custom filters: | https://raw.githubusercontent.com/Yuki2718/adblock2/main/japanese/jpf-plus.txt,<br>https://raw.githubusercontent.com/Yuki2718/adblock2/main/japanese/jpfp-ag.txt
Userscripts: | https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.meta.js<br>https://userscripts.adtidy.org/release/disable-amp/1.0/disable-amp.meta.js

### PR DESCRIPTION
**Pull request by grumaxxx**
Fix #180841

### CODE REVIEW

**Comment by grumaxxx** [comment]
I have a question about the priority of the rules. Is it better to place the rules in the language filter? Or in SocialFilter?

**Comment by grumaxxx** [comment]
And about possible analytics. How can I determine if this request is `https://c2-ap.piano.io/xbuilder/experience/execute ?aid=G5gxKHxqpj` as such?

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -60,6 +60,9 @@ gundamlog.com###more > iframe[src^="https://richlink.blogsys.jp"]
 !
 !
 !
+enterprisezine.jp##.c-article_sns
+enterprisezine.jp##.c-article_function_sns
+enterprisezine.jp##button[class="c-article_popup_btn"]
```
Comment (piquark6046) [inline JapaneseFilter/sections/specific.txt:65]: Please move to `SocialFilters`.

**Review by piquark6046 — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (grumaxxx) [reply JapaneseFilter/sections/specific.txt:65]: That was my question above. It turns out that their purpose is more important than their language affiliation, thank you!

Comment (piquark6046) [reply JapaneseFilter/sections/specific.txt]: I meant everything that you made change.

Comment (grumaxxx) [reply JapaneseFilter/sections/specific.txt]: I moved one rule to popups (which blocks it), the other two to specific

**Comment by AdamWr** [comment]
There are share buttons which are not blocked on desktop version.
<details><summary>Screenshot</summary>

image

</details>

@grumaxxx Could you please also add:
```adblock
enterprisezine.jp##.c-article_header_sns_list
```
to https://github.com/AdguardTeam/AdguardFilters/blob/master/SocialFilter/sections/specific.txt

**Review by AdamWr — APPROVED** [review]

**Comment by AdamWr** [comment]
> And about possible analytics. How can I determine if this request is `https://c2-ap.piano.io/xbuilder/experience/execute ?aid=G5gxKHxqpj` as such?

I'm not sure about it.
As far as I can see, there is a rule in annoyance filter which, as far as I understand, fixes some annoying popup on specific website, so I guess that it might be used not only for analytics.

**Pull request merged**